lamindb/migrations/0092_historylock.py

```diff
@@ -0,0 +1,30 @@
+# Generated by Django 5.1.7 on 2025-04-10 16:51
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("lamindb", "0091_history"),
+    ]
+
+    operations = [
```

Comment:
Can you just move this into the same migration file, but I can do this when refactoring the PR. ☺️